src/Moderation/Entity/Report.php

```diff
@@ -21,51 +21,125 @@
 
 namespace Friendica\Moderation\Entity;
 
+use Friendica\Moderation\Collection;
+
 /**
- * @property-read int            $id
- * @property-read int            $reporterId
- * @property-read int            $cid
- * @property-read string         $comment
- * @property-read string|null    $category
- * @property-read bool           $forward
- * @property-read array          $postUriIds
- * @property-read int            $uid
- * @property-read \DateTime|null $created
+ * @property-read int                     $id
+ * @property-read int                     $reporterCid
+ * @property-read int                     $cid
+ * @property-read int                     $gsid
+ * @property-read string                  $comment
+ * @property-read string                  $publicRemarks
+ * @property-read string                  $privateRemarks
+ * @property-read bool                    $forward
+ * @property-read int                     $category
+ * @property-read int                     $status
+ * @property-read int|null                $resolution
+ * @property-read int                     $reporterUid
+ * @property-read int|null                $lastEditorUid
+ * @property-read int|null                $assignedUid
+ * @property-read \DateTimeImmutable      $created
+ * @property-read \DateTimeImmutable|null $edited
+ * @property-read Collection\Report\Posts $posts
+ * @property-read Collection\Report\Rules $rules
  */
-class Report extends \Friendica\BaseEntity
+final class Report extends \Friendica\BaseEntity
 {
+	const CATEGORY_OTHER = 1;
+	const CATEGORY_SPAM = 2;
+	const CATEGORY_ILLEGAL = 4;
+	const CATEGORY_SAFETY = 8;
+	const CATEGORY_UNWANTED = 16;
+	const CATEGORY_VIOLATION = 32;
+
+	const CATEGORIES  = [
```

Comment:
Is `OTHER` intentionally not in the list?

Reply:
Nope, good catch.

Reply:
Added the missing one.